.github/ISSUE_TEMPLATE/ask-a-question.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Ask a question
+about: Something is not clear
+title: ""
+labels: ""
+assignees: ""
+---
+
+This should only be used in very rare cases e.g. if you are not 100% sure if something is a bug or asking a question that leads to improving the documentation. For governance related questions use https://gov.river.build/ forum.
```

Comment:
this comment makes me wonder if we should even have this template. Can we add this later if forum isn't sufficient and there is a strong signal from the community to create a ticket to "ask a question".

Reply:
Yeah i thought it might not be needed but the governance forum is more focused on gov related questions rather than code, technical questions which I do believe should be colocated with the github repo in the Issues queue.

Reply:
are we going to have a dev forum on `towns`? Like how we use team-engineering for discussing technical questions, and get fast resolution from any knowledgeable dev. I feel like filing a 'ticket' to ask a dev question is inefficient and slow to get a respond. 
If this is an RFC, then it makes sense to me to have it in github.

Reply:
Good point - I believe there's a lot of overlap between RFC and feature request, but we may want to introduce an RFC template soon. We can divert questions to the dev town once we've established one. For now, my opinion is that asking a question from the same place that you are observing the codebase is simple enough bar the additional energy needed to click a button to file a question.